### ISSUE TITLE
DMR missing in 1.23 economy files

### ISSUE THREAD

**Issue by schwitterdev** [BohemiaInteractive/DayZ-Central-Economy#24]
am i missing it or is the dmr and the new shovel not in the latest release https://github.com/BohemiaInteractive/DayZ-Central-Economy/releases/tag/DZ_123

**Comment by SirScruff** [comment]
I found this to be a 1.22 copy release.
went to nitrado and downloaded update files server side as they came with m14

**Comment by schwitterdev** [comment]
> I found this to be a 1.22 copy release. went to nitrado and downloaded update files server side as they came with m14

yeah, luckily the server was releases with a newer set of files, i was able to take these to make a diff for my server.
